src/coordination/raft_state.cpp

```diff
@@ -163,6 +163,18 @@ auto RaftState::AddCoordinatorInstance(CoordinatorToCoordinatorConfig const &con
   }
 }
 
+auto RaftState::CoordLastSuccRespMs(uint32_t srv_id) -> std::chrono::milliseconds {
+  using std::chrono::duration_cast;
+  using std::chrono::microseconds;
+  using std::chrono::milliseconds;
+
+  auto const peer_info = raft_server_->get_peer_info(static_cast<int>(srv_id));
+  auto const elapsed_time_us = microseconds(peer_info.last_succ_resp_us_);
+  spdlog::trace("Elapsed time in microseconds since last successful response from coordinator_{}: {}",
```

Comment:
Can we change this to milliseconds also?

Reply:
yes